project_task_materials_stock/models/project.py

```diff
@@ -6,6 +6,23 @@
 from openerp import models, fields, api, exceptions, _
 
 
+class ProjectProject(models.Model):
+    _inherit = 'project.project'
+
+    location_id = fields.Many2one(
```

Comment:
Can we name this `location_source_id`?
I've been using `location_id` in Tasks for the location where the work is to be done.
There is no actual collision, but using the same name for different things can be confusing.